src/mmw/js/src/data_catalog/views.js

```diff
@@ -103,6 +103,10 @@ var DataCatalogWindow = Marionette.LayoutView.extend({
         'change:detail_result': 'onDetailResultChange'
     },
 
+    onBeforeDestroy: function() {
+        App.map.set('dataCatalogDetailResult', null);
```

Comment:
I think it'd be more appropriate to add it [here](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/js/src/data_catalog/controllers.js#L64)

Reply:
Sounds good. I pushed a fixup to address this, but it seems like it hasn't shown up on this PR yet.